examples/FHIR-to-tables/FHIR-to-tables.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FHIR to Tables\n",
+    "\n",
+    "This notebook provides a tutorial on how to use functionality from the PHDI_building_blocks library to achieve the following:\n",
+    "1. Read a user-defined schema config file that specifies one or more tables.\n",
+    "2. Query a FHIR server to extract the data specified in the schema.\n",
+    "3. Transform the data into a structured format.\n",
+    "4. Load the data into one of the supported persistent tabular file formats.\n",
+    "5. Print a summary of the tables that were created.\n",
+    "\n",
+    "### Getting Started\n",
+    "\n",
+    "To make it easier to get started we have included two additional files in this direcotry:\n",
+    "1. A simple example schema config file called `example_schema.yaml`.\n",
+    "2. A `requirements.txt` file that can be used to install the `PHDI_building_blocks` library in your Python environment by running `pip install -r requirements.txt`.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from phdi_building_blocks.schemas import (\n",
+    "    make_schema_tables,\n",
+    "    print_schema_summary,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Required Parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "schema_path = \"example_schema.yaml\"  # Path to a schema config file.\n",
+    "output_path = \".\"                    # Path to directory where files will be written\n",
+    "output_format = \"parquet\"            # File format of tables\n",
+    "fhir_url = \"your_fhir_url\"           # The URL for a FHIR server\n",
+    "access_token = \"your_access_token\"   # Access token for authentication with FHIR server."
```

Comment:
At some point we'll need to pass in a credential manager, and use az login to sign in the user rather than supplying an access token.  We could prepare for that by adding a cell to do az login and change this code to instantiate a credential manager and obtain a token from it rather than expecting a token directly.  But, we can also wait until this becomes necessary.

Reply:
Added an additional job in the Test workflow to run [this](https://github.com/ResearchSoftwareActions/EnsureCleanNotebooksAction) git hub action that ensures notebooks have clean metadata. I have confirmed that `nb-clean` is capable of cleaning notebook metadata and that its use results in a notebook that was previously failing the new test to pass it.

Reply:
After discussing with @spencer-skylight these changes will be included in [PR 128](https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/138)